### PR TITLE
docs: gpbackup - add support for single level partitioned table w/ ex…

### PR DESCRIPTION
**Pull request by mkiyama**
…t. tbl. leaf partition.

PR for 5X_STABLE
Will be ported to MAIN

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/managing/backup-gpbackup.xml [hunk 1]
```diff
@@ -207,9 +207,9 @@ $ <b>gprestore -backupdir /home/gpadmin/backups/ -timestamp 20171103153156 -crea
 20171103:15:51:17 gprestore:gpadmin:0ee2f5fb02c9:017819-[INFO]:-Post-data metadata restore complete</codeblock></p>
       <p><codeph>gprestore</codeph> does not attempt to restore global metadata for the Greenplum
         System by default. If this is required, include the <codeph>-globals</codeph> argument.</p>
-      <p>By default, <codeph>gprestore</codeph> uses 2 parallel connections to restore table data.
```
Comment (khuddlefish) [inline gpdb-doc/dita/admin_guide/managing/backup-gpbackup.xml]: Did you push this commit to the wrong PR? I think we have another one open for this content.

**Comment by mkiyama** [comment]
@khuddlefish @chrishajas 
hmmm... to many related PRs
For this change.
gprestore --jobs option default is 1.

Chris had this comment on a different PR, but the PR was already merged.
https://github.com/greenplum-db/gpdb/pull/3899#discussion_r151491096

I thought the comment was for this PR. Is the information incorrect?

File: gpdb-doc/dita/admin_guide/managing/backup-gpbackup.xml [hunk 2]
```diff
@@ -205,9 +207,9 @@ $ <b>gprestore -backupdir /home/gpadmin/backups/ -timestamp 20171103153156 -crea
...
-        If you have a large backup set you can improve performance of the restore option by
-        increasing the number of parallel connections with the <codeph>-jobs</codeph> option. For
+      <p>By default, <codeph>gprestore</codeph> uses 1 connection to restore table data. If you have
+        a large backup set you can improve performance of the restore option by increasing the
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/backup-gpbackup.xml]: Remove first "option" in this sentence.

**Comment by khuddlefish** [comment]
Ok. I didn't see that the other PR was merged. Both commits in this PR look correct to me. Both the docs about single level partition and the change for default jobs to 1.

**Comment by mkiyama** [comment]
@dyozie 
I think this can be merged.

**Comment by dyozie** [comment]
Thanks - will do.

**Pull request merged**